src/main/java/utils/WaitUtils.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class WaitUtils {
+    public static void waitUntilElementsIsPresented(WebDriver driver, By locator, int durationInSeconds) {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
```

Comment:
Why 10 is passed to Duration.ofSeconds(10) ? Should be method argument used: Duration.ofSeconds(durationInSeconds)

If you don't want to pass every single time this durationInSeconds, the argument should be removed.